Gruntfile.js

```diff
@@ -29,12 +35,36 @@ module.exports = function (grunt) {
                 enableJSON5: true
             },
             src: ["gpii/**/*.json5", "tests/**/*.json5", "testData/**/*.json5", "*.json5"]
+        },
+        mdjsonlint: {
+            src: ["./*.md", "./documentation/**/*.md", "./examples/**/*.md"]
+        },
+        markdownlint: {
```

Comment:
Can this go in a config file or be shared somehow?

Reply:
I am experimenting with that at the moment, my hope is that we can do so, yes.